README.md

```diff
@@ -33,5 +39,33 @@ Open up [http://localhost:3000](http://localhost:3000) and you should be ready t
 
 If you're used to using the `vercel dev` command provided by [Vercel CLI](https://vercel.com/cli) instead, you can also use that, but it's not needed.
 
-
 <img align="center" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSagMR3FC2rPHvoqZFjHunC5DkmFLdeNzok9d7P8OHW&s" width="50%" alt="Vercel" >
+
+## Contributing - [look at our doc here](./CONTRIBUTING.md)
```

Comment:
Should we move this section into the `CONTRIBUTING.md` file? What do you think?

Reply:
Yeah for sure, was trying to think of a way to direct people from the readme to the contributing markup. Not entirely sure if you have to tell people where to go or if they just know.

Reply:
I would recommend we bundle all "Contribution" information in one place, so it's easier to find. This could be a section in the `README` or a standalone `CONTRIBUTION` document. If we have a separate document, I would refer to it in the `README`! :)

Reply:
Ok! I prefer less files. Happy to put back in if the group misses CONTRIBUTING.md